packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -32,6 +32,128 @@ import 'scrollable.dart';
 /// that a [SingleChildScrollView] containing a [ListBody] or [Column] with
 /// many children.
 ///
+/// ## Sample code: Using [SingleChildScrollView] with a [Column]
+///
+/// Sometimes a layout is designed around the flexible properties of a [Column],
+/// but there is the concern that on some devices, there might not be enough
```

Comment:
This is often the case because landscape mode.

Reply:
changed to:
```
/// Sometimes a layout is designed around the flexible properties of a
/// [Column], but there is the concern that in some cases, there might not
/// be enough room to see the entire contents. This could be because some
/// devices have unusually small screens, or because the application can
/// be used in landscape mode where the aspect ratio isn't what was
/// originally envisioned, or because the application is being shown in a
/// small window in split-screen mode. In any case, as a result, it might
/// make sense to wrap the layout in a [SingleChildScrollView].
```